doc/release-notes/11744-cors-echo-origin-vary.md

```diff
@@ -0,0 +1,33 @@
+# 11744: CORS handling improvements
+
+Modernizes CORS so browser integrations (previewers, external tools, JS clients) work correctly with multiple origins and proper caching.
+
+## Highlights
+- Echoes the request origin (`Access-Control-Allow-Origin`) when it matches `dataverse.cors.origin`.
+- Adds `Vary: Origin` for per-origin responses (not for wildcard).
+- Supports comma‑separated origin list; any `*` in the list = wildcard mode.
+- CORS now only enabled when `dataverse.cors.origin` is set (deprecated `:AllowCors` no longer enables it).
+- Allows readable spacing in CORS list settings (`dataverse.cors.methods`, `dataverse.cors.headers.allow`, `dataverse.cors.headers.expose`): spaces around commas are ignored; tokens are otherwise unchanged (no quote parsing).
```

Comment:
This is true for all database and microprofile settings tat accept comma-separated values - probably should be it's own top-level entry rather than just part of the CORS entry.